components/count-badge/README.md

```diff
@@ -24,6 +25,8 @@ The `d2l-count-badge` element is a web component to display a number count, eith
 | `type`, default: `count` | String | The type of the badge. Valid options are `"notification"` and `"count"`. Notification badges are orange and count badges are grey. |
 | `max-digits`, default: `2` when `type="notification"` | Number | Optionally specify a digit limit, after which numbers are truncated. Defaults to two for `"notification"` type and no limit for `"count"` type.
 | `hide-zero`, default: `false` | Boolean | Optionally choose not to show the count badge when the number is zero. |
+| `icon`, default: no icon | String | Optionally choose an icon to render the badge above. |
+| `icon-size`, default: none | Boolean | Optionally choose a fixed height/width for the icon. Defaults to icon's respective tier size. |

```

Comment:
This is `fixed-icon-size`, ya? Could you give some more detail about the scenarios for which the default tier size isn't good enough? Just trying to think of other ways around this.

Reply:
Yep, this comment was from before I moved it into a private property. The one case we have to have this for is d2l-card-footer link, which forces all icons to 18*18. That's really the only case, which consumers of the card-footer-link don't have to worry about. That's why I wanted to sort of hide it in a private method, it should really only be used in that case where the icons must be fixed for formatting purposes.

Reply:
Would we be able to audit the uses of the card footer link and just make sure that they're all tier1 icons and if not, switch them to tier1 icons?

It's just that we don't do this kind of thing anywhere else (e.g. `<d2l-button-icon>`, `<d2l-button-subtle>`) -- we just assume that people are using tier1 icons with them and if they aren't, things look bad.

Reply:
Looks like they are all tier1 icons, you're right. That will simplify the code a bit!